src/CodeGeneration.Roslyn.Engine/CodeGeneration.Roslyn.Engine.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard1.6</TargetFramework>
+    <LangVersion>7.3</LangVersion>
+    <SignAssembly>True</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\opensource.snk</AssemblyOriginatorKeyFile>
+    <Description>The engine of source code generation used by `dotnet-codegen` tool.</Description>
```

Comment:
So I guess this creates a new nuget package. Is that right? And does the distribution all work out, including with the `dotnet-codegen` tool?

Reply:
Yes, I can build a generator-using project from local NuGet feed with these artifacts.

In terms of pushing this new package to official NuGet or MyGet feeds, I don't know how that'll work.